pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/osteo/OsteoV2ReleasePdfUpdates.java

```diff
@@ -0,0 +1,104 @@
+package org.broadinstitute.ddp.studybuilder.task.osteo;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dao.JdbiUser;
+import org.broadinstitute.ddp.db.dao.PdfDao;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.db.dto.UserDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.pdf.PdfConfigInfo;
+import org.broadinstitute.ddp.model.pdf.PdfVersion;
+import org.broadinstitute.ddp.studybuilder.PdfBuilder;
+import org.broadinstitute.ddp.studybuilder.task.CustomTask;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.List;
+
+@Slf4j
+public class OsteoV2ReleasePdfUpdates implements CustomTask {
+
+    private static final String DATA_FILE = "patches/osteo-v2-release-pdfs.conf";
+    private static final String STUDY_GUID = "CMI-OSTEO";
+
+    private Path cfgPath;
+    private Config studyCfg;
+    private Config dataCfg;
+    private OsteoV2ReleasePdfUpdates.SqlHelper sqlHelper;
+
+    @Override
+    public void init(Path cfgPath, Config studyCfg, Config varsCfg) {
+        if (!studyCfg.getString("study.guid").equals(STUDY_GUID)) {
+            throw new DDPException("This task is only for the " + STUDY_GUID + " study!");
+        }
+        this.cfgPath = cfgPath;
+        this.studyCfg = studyCfg;
+
+        File file = cfgPath.getParent().resolve(DATA_FILE).toFile();
+        if (!file.exists()) {
+            throw new DDPException("Data file is missing: " + file);
+        }
+        this.dataCfg = ConfigFactory.parseFile(file).resolveWith(varsCfg);
+    }
+
+    @Override
+    public void run(Handle handle) {
+        this.sqlHelper = handle.attach(OsteoV2ReleasePdfUpdates.SqlHelper.class);
+        StudyDto studyDto = handle.attach(JdbiUmbrellaStudy.class).findByStudyGuid(studyCfg.getString("study.guid"));
+        UserDto adminUser = handle.attach(JdbiUser.class).findByUserGuid(studyCfg.getString("adminUser.guid"));
+
+        List<? extends Config> pdfs = dataCfg.getConfigList("pdfs");
+        pdfs.forEach(pdf -> cleanupV2Version(handle, pdf.getString("name"), studyDto.getId(), "v2"));
+
+        //re-insert new v2 version
+        var pdfBuilder = new PdfBuilder(cfgPath.getParent(), studyCfg, studyDto, adminUser.getUserId());
```

Comment:
I'm noticing an intermixing of `var` and the old-style type definitions- can we consistently use one or the other?

Reply:
Patches get lot of copy & paste code because of one time run. Must be side effect of it. :grin: 
Will clean up Blake.

Reply:
Thank you!